.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!--
+
+Do you want to ask a question? Are you looking for support? The SAML CTK
+developers group at https://groups.google.com/forum/#!forum/saml-ctk
+is the best place for getting support.
+
+By filing an Issue, you are expected to comply with the SAML CTK's Code of Conduct.
+See https://github.com/codice/saml-conformance/blob/master/.github/CODE_OF_CONDUCT.md
+
+-->
+### Description
+
+[Description of the issue]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+#### Expected behavior:
+[What you expect to happen]
+
+#### Actual behavior:
+[What actually happens]
+
+#### Reproduces how often:
+[What percentage of the time does it reproduce?]
```

Comment:
✏️ This wording sounds weird to me. Maybe "Occurs how often" and "What percentage of the time does it occur"?